packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -480,6 +490,10 @@ - (void)keyDown:(NSEvent *)event
 		[[self window] makeFirstResponder:viewToFocus];
 		[viewToFocus scrollRectToVisible:[viewToFocus bounds]];
 	}
+	else if (noOp)
+	{
+		// No view to focus, do nothing
```

Comment:
Out of curiosity, if in the noOp case (`nextViewToFocus == nil`) we called `[super keyDown], what happens?

Reply:
In the case of FocusZoneActionTab, calling `[super keyDown]` would move focus to the next menu item (the next valid key view?)

Reply:
I see, thanks!